System.IO.Streams/MemoryStream.cs

```diff
@@ -459,6 +471,18 @@ private void EnsureOpen()
             }
         }
 
+        /// <summary>
+        /// Check that stream is writable
+        /// </summary>
+        /// <exception cref="NotSupportedException"></exception>
```

Comment:
can you please move this comment to all the functions where the EnsureWritable() is used? And also add a little comment saying that it throw if the stream is not writable or something equivalent?

Reply:
I'll get that in later today.